### PR TITLE
chore(ci-lint): Add some markdown lint rules

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Adds some linter rules based on https://github.com/cloudquery/cloudquery/pull/5647 and https://github.com/cloudquery/cloudquery/pull/5643

### CODE REVIEW

File: .github/styles/proselint/RealWorld.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extends: existence
+message: "Consider removing '%s'."
+ignorecase: true
+level: warning
+tokens:
+  - 'real world'
```
Comment (erezrokah) [inline .github/styles/proselint/RealWorld.yml:6]: We don't need to say `real world examples`

Comment (disq) [reply .github/styles/proselint/RealWorld.yml:6]: lol

Comment (erezrokah) [reply .github/styles/proselint/RealWorld.yml:6]: Saving the review for my future self

Comment (erezrokah) [reply .github/styles/proselint/RealWorld.yml:6]: giphy

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**